docs/pages/hdmi-cec-sink-L3-Low-Level_TestSpec.md

```diff
@@ -0,0 +1,346 @@
+  - [Initialization and Startup]
+
+## Overview
+This document provides an overview of the L3 testing requirements for the module `HDMI CEC` source device. It outlines the scope of testing, objectives, External devices, specific test requirements,  expected deliverables, and approaches to satisfy the L3 Test requirement.  
+
+## Acronyms, Terms and Abbreviations
+- `CEC`    -  Consumer Electronics Control
+- `HAL`    -  Hardware Abstraction layer
+- `HDMI`   -  High Definition Multimedia Interface
+- `API`    -  Application Program Interface
+- `SoC`    -  System on Chip
+- `DUT`    -  Device Under Test
+- `LA`     -  Logical Address 
+- `PA`     -  Physical Address
+
+
+## References
+- CEC Adaptor: https://www.pulse-eight.com/p/104/usb-hdmi-cec-adapter#
+- `HAL` interface file: https://github.com/rdkcentral/rdk-halif-hdmi_cec/blob/main/include/hdmi_cec_driver.h
+
+
+## Module Description
+Consumer Electronics Control (`CEC`) is a one-wire bidirectional bus within an HDMI system that enables interconnected products to communicate. This module outlines a set of `HAL` `APIs` designed to support `HDMI` `CEC` functionality. The current document details the L3 test definitions and descriptions used to validate these `HDMI` `CEC` `APIs`, including the prerequisites that must be managed before testing and the external stimuli that need to be applied during the test.
+
+The present document describes the test scope for the Sink Device activities only.
+
+### HDMI-CEC L3 Test Functionality
+The picture below depicts the HDMI CEC L3 Test Functionality Setup. This simple setup has an HDMI CEC Pulse-eight USB adaptor, which acts as a source device and can send the required commands (using libcec tool commands) and respond to the received CEC Commands.  
+
+Note: The below-shown prerequisites should be met before starting the test on the platforms. 
+- All the devices used in the test setup should support the `HDMI` `CEC` feature during the entire test duration.
+- HDMI drivers should be up and running on the platform before running this test.
+- For now, it is suggested not to connect multiple CEC Adaptors onto a single PC as there is no proper support on libcec for multiple devices.
+
+```mermaid
+graph TB
+C[PC] <--> A
+A[Pulse-Eight CEC Adaptor ] <--> |HDMI| B[TV]  
+```
+
+### Pulse-Eight CEC Adaptor tool:
+The Pulse-Eight CEC Adapter will be used to frame and send commands to the DUT. This tool leverages libcec, which can be installed to facilitate CEC activities during testing. The cec-client tool will be employed extensively throughout the test. Additionally, RAFT can use this tool to automate the test cases.
+
+- Reference to the tool: https://www.pulse-eight.com/p/104/usb-hdmi-cec-adapter
+- libcec tools: https://www.pulse-eight.com/Download/Get/51  
+
+**Libraries and tools required for RAFT**
+- libcec should be installed on the PC where the Pulse-Eight CEC Adaptor is connected.
```

Comment:
libcec should not be required, because the messages are deterministic

Reply:
libcec is the main software used to control the adaptor.  Else, we should specify what one needs to install here.